.evergreen/config.yml

```
@@ -1035,6 +1047,22 @@ buildvariants:
   - name: publish-packages
     distros:
     - rhel70-small
+- name: debian11
+  display_name: "Debian 11.0"
+  run_on: debian11-large
```

```suggestion
  run_on: debian11-small
```

Suggest using small since libmongocrypt builds are not resource intensive.

---

So, when I looked in the Evergreen host listings, I found that there are 4 debian11-large hosts and 0 debian11-small.  With the other distros it looks like there are a few -large and considerably more -small hosts.  I'm not sure why there aren't any debian11-small hosts, but I think the build would just stall if we went that route.  Either that, or debian11-small is an alias to debian11-large, meaning that there is only one size of host available for debian11 builds.

---

Interesting. There is a task in the Go driver using debian11-small: https://spruce.mongodb.com/task/mongo_go_driver_testgcpkms_variant_testgcpkms_fail_task_0e5ae89f1baf3d6edca093467ffd1f870b441add_22_11_28_21_11_31/logs?execution=0. Maybe that is an alias.